apps/web/app/actions.ts

```
@@ -2,118 +2,200 @@
+	try {
+		const { error } = await supabase.auth.signOut()
+		if (error) {
+			return errorHandler.handleAuthError(error, 'sign_out')
+		}
+
+		return {
+			success: true,
+			message: 'Successfully signed out',
+			redirect: '/sign-in',
+		}
+	} catch (error: any) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)